### ISSUE TITLE
(OrcaSlicer 1.6.3) Travel speed very slow

### ISSUE THREAD

**Issue by UsSuggester** [SoftFever/OrcaSlicer#1415]
I set the travel speed up to **200mm/s** but on every layer, I get a real travel speed of about **20mm/s.**

Anyway, on the graphics, slicer says that the travel speed is the one I set.

Suposition-Question 1:
_-It is true that I set spiral Z Hop and the printer is doing an slope every time since I move to Klipper (more speed, maybe some speed or acceleration limits are giving me troubles with spiral ZHop and causing also this slow speed while traveling, I will check._

Suposition-Question 2:
_-Could this be a conflict between travel speed and "min print speed" on Part Cooling Fan? (It is the only speed parameter that I set with a very slow or similar value to the speed observed on traveling)._

**Printer model**
Ender 3 S1 (on Marlin and Klipper)

**Desktop (please complete the following information):**
I'm on MacOS 13.4

**Comment by blueeagle69** [comment]
I appear to have noticed the same issue.
I have my travel speed set to 180mm/s and my Initial layer travel speed set at 100%, but when it's printing, the nozzle is moving painfully slow. Including before the print starts. Prime line etc.

**Comment by secretstanxxx** [comment]
how did you fix this

**Comment by deemoss** [comment]
Has I get the same thing

**Comment by Sate04** [comment]
was this ever solved? I'm having the same issue right now, I don't want to switch back to SuperSlicer but thats what I'm having to use right now.

**Comment by deemoss** [comment]
> was this ever solved? I'm having the same issue right now, I don't want to switch back to SuperSlicer but thats what I'm having to use right now.

For me this was solved by creating a new printer profile from scratch.

**Comment by Sate04** [comment]
Found the issue, OP was onto something with the spiral Z-Hop, I had copied it from the X1C slicer settings. Turning that off fixed my travels. Z-Hop is hidden in the printer profile extruder, making a new profile would've gotten rid of that, probably why it fixed the issue for you.

**Comment by RTVRTV** [comment]
Under Filament settings --> Tab Cooling --> Turn off "Slow printing down for better layer cooling"

**Comment by CodeMasterCody3D** [comment]
I know how to fix slow travel while spiral z hop. 
Recompile firmware, look for arc moves enable it, look for CNC_WORKSPACE. (ARC MOVES WILL BE ABOVE IT) I also enabled P moves or what ever it's called. I haven't had sleep been working on this for days. Your welcome. Spiral z hop for the win guys. Imma have a video out soon with a tutorial. Also I had to set jerk limits for I and J moves to 20 so it actually spins fast. Also my travel speed is 300mm/s ender 3 pro using Bambu hot end, bambu gcode for prime line and modified firmware for CNC workspace, arc moves, jerk is set to 20 for I and J moves (it's used for making the circle. P is also used in the gcode spiral z hop as well and I noticed it while scrolling the firmware. Like I said I'll have a video out soon. Sorry for vague information.
